guardian/ophan-housekeeper
# Update Housekeeper code and Readme for local testing

## What does this change?

We upgraded Ophan Housekeeper from Scala 2 to Scala 3  #257 and that means the `main` function must be changed because it is not the same in Scala 3 and this is how we can run locally.

## How to test

Run locally to see if the changed code works as expected. For more information check the updated Readme. 

## How can we measure success?

We are able to test locally.

## Have we considered potential risks?

When running locally the developer creates an email alert on their own account so that shouldn't cause any alerts deletion except for the developer.
